config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -483,7 +483,8 @@ presubmits:
         env:
         - name: SKIP_RELEASE_VALIDATION
           value: "true"
-        image: gcr.io/k8s-testimages/kubekins-e2e:v20190201-234c5ddcf-master
+        image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
+        imagePullPolicy: always
```

Comment:
`always` should be uppercase?
https://github.com/kubernetes/test-infra/blob/master/config/tests/jobs/jobs_test.go#L540

Reply:
yes :man_shrugging: